src/ray/protobuf/node_manager.proto

```diff
@@ -51,6 +51,16 @@ message RequestWorkerLeaseRequest {
 }
 
 message RequestWorkerLeaseReply {
+  // Request lease worker cancel type.
+  enum CancelType {
+    // Canceled because of the removal of the corresponding placement group.
```

Comment:
@rkooo567 , I decided not to remove the field `canceled` temporarily cause it will involve too much other modification, I will remove it in a separate pr.